packages/db/src/workspace/definitions.ts

```diff
@@ -1,37 +1,40 @@
-import { Definitions } from "./pouch";
+import * as Meta from "@truffle/db/meta";
+import * as Pouch from "./pouch";
 
-export type WorkspaceCollections = {
+export type Collections = {
```

Comment:
I'm a little iffy on the renaming here, because the `meta` file also has a `Collections` type.  I guess `WorkspaceCollections` won't work anymore once it is the root schema, but wondering if there is a better naming option here. Or am I misunderstanding how this works?

Reply:
I'm not so worried about the name collision? Maybe that's just my style. Nothing will ever import both - if necessary, I'm thinking we do `import * as Meta from "@truffle/db/meta"`